feeluown/utils/reader.py

```diff
@@ -139,8 +139,9 @@ def read_next(self):
             try:
                 obj = next(self._g)
             except StopIteration:
+                # let count=offset, or the last adjust_height not work
                 if self.count is None:
-                    self.count = self.offset + 1
+                    self.count = self.offset
```

Comment:
I think we should the root cause is the L229. It should be `count > offset + 1`.
https://github.com/feeluown/FeelUOwn/blob/e72935c61d3fe5ef1af7d049acf6bbf52c674bb2/feeluown/gui/helpers.py#L224-L233